packages/Meshfree/src/DTK_MovingLeastSquaresOperator_def.hpp

```diff
@@ -59,8 +58,6 @@ MovingLeastSquaresOperator<DeviceType, CompactlySupportedRadialBasisFunction,
     source_points = Details::NearestNeighborOperatorImpl<DeviceType>::fetch(
         _comm, _ranks, _indices, source_points );
 
-    // TODO: ensure that we found enough source points to proceed
```

Comment:
Where is this done?

Reply:
No where since we decided on Tuesday that any output is good enough...

Reply:
Let's add a test with an empty source cloud in `tstMovingLeastSquaresOperator.cpp` in that case

Reply:
There is already such a test.